integreat_cms/deepl_api/deepl_api_client.py

```diff
@@ -141,6 +141,17 @@ def translate_queryset(self, queryset, language_slug):
                     target_language.slug
                 )
 
+                # Before translating, check if translation is not up-to-date
+                if existing_target_translation.translation_state in (
```

Comment:
```suggestion
                if existing_target_translation and existing_target_translation.translation_state in (
```

To avoid `Nonetype Object` error, check existence of translation first then status.

Reply:
Oh, that's a very good point. Thank you!